pkg/consts/consts.go

```diff
@@ -278,6 +278,10 @@ const (
 	// is `a=b,c=d,...`. After updated, the old user-assigned tags would not be replaced by the new ones.
 	ServiceAnnotationAzurePIPTags = "service.beta.kubernetes.io/azure-pip-tags"
 
+	// ServiceAnnotationLoadBalancerFloatingIP is the annotation used on the service to enable/disable floating IP in load balancer rule.
+	// If omitted, the default value is true.
+	ServiceAnnotationLoadBalancerFloatingIP = "service.beta.kubernetes.io/azure-load-balancer-floating-ip"
```

Comment:
since floating ip is enabled by default, maybe it's better rename it to "service.beta.kubernetes.io/azure-load-balancer-disable-floating-ip"?

Reply:
The field name in LB rule is "EnableFloatingIP" in azure sdk. How does "service.beta.kubernetes.io/azure-load-balancer-enable-floating-ip" sound just to keep consistent with the wording?